demo/body3d_two_stage_video_demo.py

```diff
@@ -145,6 +153,36 @@ def main():
         default='configs/_base_/filters/one_euro.py',
         help='Config file of the filter to smooth the pose estimation '
         'results. See also --smooth.')
+    parser.add_argument(
+        '--return-heatmap',
+        action='store_true',
+        default=False,
+        help='whether to return heatmap')
+    parser.add_argument(
+        '--output-layer-names',
+        nargs='*',
+        type=str,
+        help='return the output of some desired layers, '
+        'e.g. use ("backbone", ) to return backbone feature')
+
+    parser.add_argument(
+        '--use-multi-frames',
+        action='store_true',
+        default=False,
+        help='whether to use multi frames for inference. Default: False.')
+    parser.add_argument(
+        '--online',
+        action='store_true',
+        default=False,
+        help='inference mode. If set to True, can not use future frame'
+        'information when using multi frames for inference. Default: False.')
```

Comment:
Please specify that these arguments are for the pose detection stage in the argument names and help messages.

Reply:
OK, I will change it.